src/aws/osml/tile_server/main.py

```diff
@@ -32,18 +39,14 @@
     logging.DEBUG: "debug",
 }
 
-# Configure logger
-logger = logging.getLogger("uvicorn")
-logger.setLevel(ServerConfig.tile_server_log_level)
 
 try:
     aws_ddb, aws_s3, aws_sqs = initialize_aws_services()
 except ClientError as err:
-    logger.error(f"Fatal error occurred while initializing AWS services. Exception: {err}")
-    sys.exit("Fatal error occurred while initializing AWS services. Exiting.")
+    sys.exit(f"Fatal error occurred while initializing AWS services. Exiting. {err}")
 
 
-def initialize_viewpoint_components() -> Tuple[ViewpointStatusTable, ViewpointRequestQueue, ViewpointRouter]:
+def initialize_viewpoint_router() -> ViewpointRouter:
     """
     Initialize viewpoint-related components.
```

Comment:
Update to `Initialize viewpoint router`

Reply:
Update the docstring to reflect on the new return type

Reply:
fixed